File: components/Blank/ts/Component.ts
import {html, LitElement, unsafeCSS} from 'lit';
import {customElement} from 'lit/decorators.js';
// @ts-ignore
import styles from '../scss/Styles.scss';

@customElement('zn-blank')
export class ZincIcon extends LitElement {
    static styles = unsafeCSS(styles);

    render() {

        return html`
          Hello <slot></slot>`;
    }
}

File: components/Blank/go/component.go
package Blank

import (
	"fmt"
	"html/template"
)


type Instance struct {
}

func New() Instance {
	return Instance{}
}

func (i Instance) Html() template.HTML {
	return template.HTML(fmt.Sprintf(`<zn-blank></zn-blank>`))
}
